### PR TITLE
Update site header to LeadsForce

### PR DESCRIPTION
**Pull request by CrownKira**
This PR fixes some formatting issues in DG and UG.

Tips: 
Make sure to run `./.github/run-checks.sh` before pushing

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 ---
-layout: page
-title: Developer Guide
+layout: page title: Developer Guide
```
Comment (wyrchris) [inline docs/DeveloperGuide.md:2]: The markdown formatting(see rich diff preview) seems to be off

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
 ---
-layout: page
-title: User Guide
+layout: page title: User Guide
```
Comment (wyrchris) [inline docs/UserGuide.md:2]: as above

File: docs/UserGuide.md [hunk 3]
```diff
@@ -202,18 +215,24 @@ If your changes to the data file makes its format invalid, AddressBook will disc
...
+the data of your previous LeadsForce home folder.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command summary
 
 Action | Format | Examples
 --------|---------|---------
-**Create** | `create {client’s name} /<email> {email} /<phone-no> {phone number} /<risk-appetite> {risk appetite}`| create benedict /email benedict@gmail.com /phone-no 90909898 /risk-appetite 3 |
+
```
Comment (wyrchris) [inline docs/UserGuide.md:227]: Formatting for table seems off

**Review by wyrchris — COMMENTED** [review]
Overall looks good other than the formatting for certain parts to check

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (CrownKira) [reply docs/DeveloperGuide.md:2]: The issue seems to be related to IntelliJ default formatter. I will look into the issue.

File: docs/UserGuide.md (hunk 3 above)
Comment (CrownKira) [reply docs/UserGuide.md:227]: The formatter doesn't support formatting the table in markdown apparently.

**Review by benedictchuajj — APPROVED** [review]
Formatting looks good. 👍  
Thanks for fixing the formatting inconsistencies and for reminding us of the use of `./.github/run-checks.sh`

**Pull request merged**